config/factions/conflux.json

```diff
@@ -152,8 +152,7 @@
 			"mageGuild" : 5,
 			"primaryResource" : "mercury",
 			"warMachine" : "ballista",
-			"moatDamage" : 70,
-			"moatHexes" : [ 11, 28, 44, 61, 77, 111, 129, 146, 164, 181 ],
+			"moatAbility" : "core:spell.castleMoat",
```

Comment:
why not `"moatAbility" : "core:spell.confluxMoat",` only `"moatAbility" : "core:spell.castleMoat",`?

Reply:
Because castle and conflux moats is identical in every aspect, even name. Initially I want to do 4 type of moat - castle, fortress, inferno, and tower. All other are defined purely for names only.